modules/soplex/7.1.2.bcr.1/overlay/BUILD.bazel

```diff
@@ -0,0 +1,53 @@
+# Description:
+# Soplex is a LP solver, a common tool in Operations Research.git
+
+cc_library(
+    name = "zstr",
+    hdrs = glob(["src/soplex/external/zstr/*.hpp"]),
+    includes = ["src/soplex/external"],
+    visibility = ["//visibility:public"],
```

Comment:
Is this really meant to be public? This could result in users depending on your module just to get zstr. Since it's header only, submitting it as a dedicated BCR module should be pretty easy.

Reply:
PR submitted for zstr

Reply:
Done.